lms/djangoapps/support/views/sso_records.py

```diff
@@ -36,5 +56,8 @@ def get(self, request, username_or_email):  # lint-amnesty, pylint: disable=miss
         except User.DoesNotExist:
             return JsonResponse([])
         user_social_auths = UserSocialAuth.objects.filter(user=user)
-        sso_records = serialize_sso_records(user_social_auths)
+        sso_records = []
+        for user_social_auth in user_social_auths:
+            user_social_auths_history = UserSocialAuth.history.filter(id=user_social_auth.id)
+            sso_records.append(serialize_sso_records(user_social_auth, user_social_auths_history))
```

Comment:
Would we face multiple-provider SSOs for a single user inconsistency with this filtering?